File: Basics.py
# 1. Variables and Data Types:

name = "Barry"  # String
age = 25        # Integer
height = 5.9    # Float
is_student = True  # Boolean


# 2. Print Output:

print("Hello, World!")  # Output: Hello, World!
print(f"My name is {name} and I am {age} years old.")  # Using f-strings


# 3. Taking Input:

name = input("What is your name? ")
print(f"Hello, {name}!")


# 4. Conditionals:

age = 18
if age >= 18:
    print("You are an adult.")
else:
    print("You are a minor.")


# 5. Loops:

#For loops:
for i in range(5): # Loops from 0 to 4
    print("Iteration:", i)

#While Loops:
count = 0 
while count < 3: 
    print(f"Count is: {count}")
       count += 1


# 6. Functions (with parameters): 
def greet(name):
    return f"Hello, {name}!"

print(greet("Barry")) # Output: Hello, Barry!


# 7. Lists(Arrays):
fruits = ["apple", "banana", "cherry"]
print(fruits[0])  # Output: apple

fruits.append("orange")  # Add an element
print(fruits)  # Output: ['apple', 'banana', 'cherry', 'orange']


# 8. Dictionaries(Key-Value-Pairs):
person = {"name": "Barry", "age": 25}
print(person["name"])  # Output: Barry

person["age"] = 26  # Update value


# 9. Loops with lists:

numbers = [1,2,3,4]
for num in numbers:
    print(num)  #Prints each number

# 10. Simple Logic

number = 4
if number % 2 == 0:
    print(f"{number} is even.")
else:
    print(f"{number} is odd.")

File: Basics.js
//1. Variables 

let name = "Barry"; // can be reassigned 
const age = 25; // cannot be reassigned 
var city = "Tampa" // Older declaration (use let/const instead)

//2. DataTypes

let number = 5; // Number
let text = "Hello, World!" //String
let isAvailable = true; //Boolean 
let list = [1,2,3 ]; //Array
let person = { name:"Barry", age:25} //Object
let nothing = null;n //Null
let unknown; //Undefined

//3. Conditionals 

let age1 = 20; 
if (age1 >= 18) {
    console.log("You are an adult.");  
} else {
    console.log("You are a minor.")

}

//4. Loops:

//For Loops:
for (let i = 0; i < 5; i++) {
    console.log("Iteration:",);
}

//While Loops:
let count = 0;
while(count < 3) {
    console.log("Count is: count");
    count++;
}

//For-of-Loops:
let numbers = [10, 20, 30];
for (let number of numbers) {
    console.log(number)
}

//5. Functions: 

//Regular Function declartion:
function greet(name) {
    return "Hello, " + name + "!";
}
console.log(greet("Barry")) // Output: Hello, Barry!

//Arrow Function:
const greet = (name) => {
    return `Hello, ${name}!`;
};
console.log(greet("Barry")); // Output: Hello, Barry!

//6. Arrays:
let fruits = ["apple", "banana", "orange"];
console.log(fruits[0]); //Output: apple

fruits.push("grape"); //Adds an element
console.log(fruits);     // Output: ['apple', 'banana', 'orange', 'grape'] 

//7. Objects:
let car = {
    brand: "Toyota",
    model: "Corolla",
    year: 2020
  };
  
  console.log(car.brand);  // Output: Toyota
  car.year = 2021;         // Update property

  